examples/player/simid_player.js

```diff
@@ -99,6 +99,18 @@ class SimidPlayer {
      */
     this.rejectInitializationPromise_ = null;
 
+    /**
+     * A boolean value that tracks whether or not the creative has been resized.
+     * @private {boolean}
+     */
+    this.isResized_ = false;
```

Comment:
Since resizeDimensions_ starts as null, I don't think you need this extra tracker variable

if (this.resizeDimensions_) should be enough to check if its been resized, since that will evaluate to false when resizeDimensions is null